webapp/publisher/snaps/build_views.py

```diff
@@ -381,15 +381,13 @@ def post_snap_builds(snap_name):
 
 @login_required
 def post_build(snap_name):
-    try:
-        details = api.get_snap_info(snap_name, flask.session)
-    except ApiResponseErrorList as api_response_error_list:
-        if api_response_error_list.status_code == 404:
-            return flask.abort(404, "No snap named {}".format(snap_name))
-        else:
-            return _handle_error_list(api_response_error_list.errors)
-    except ApiError as api_error:
-        return _handle_error(api_error)
+    details = api.get_snap_info(snap_name, flask.session)
+
+    # Don't allow builds from Admins that are no contributors
```

Comment:
@Lukewh I added this, I think we were missing it. Does make sense to check this?